Ghost Cafe Remodel

Thlumyn opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

Collaboration with @TheSS13Melon 

Changes up the ghost cafe, moving and condensing the existing cabins while adding some new areas, including a mini-station, an ashwalker camp and an ice cat camp. 

It also removes some unused cafe areas (on the code end). 

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## How This Contributes To The Skyrat Roleplay Experience

A lot of people have scenes that make sense to happen on a space station - either continuations of scenes that happened in previous rounds, or new scenes altogether. While going to the cafe and pretending that you were still in the bridge, or still in the CE office, or still in the interrogation room, was well and fine, adding simplified mini versions of those might be nice for people to stay immersed. Some of these were already sort of there, such as a surgery room, janitor room and cell that seemed to be more or less placed randomly into the cafe area. The cafe is still supposed to be a cafe however, and we tried to balance the outdoor cafe-y theme with the station parts.

This also adds areas such as a sauna/gym that people can use in the cafe. And a camp for the ice cats and ashwalkers. 

<!-- Please add a short description of why you think these changes would benefit the game and the roleplay atmosphere of the server. If you can't justify it in words, it might not be worth adding. -->

## Proof of Testing

<!-- Include any screenshots/videos/debugging steps of the code functioning successfully, between the </summary> and </details> code blocks. -->
<!-- To our mappers and spriters: Posting screenshots of content INSIDE EDITORS (aseprite, PDN, SDMM, ect) is NOT valid proof of testing. Please make sure that you COMPILE the game and provide PROOF you tested your edits. -->

<details>
<summary>Screenshots/Videos</summary>
  
![image](https://github.com/Skyrat-SS13/Skyrat-tg/assets/102194057/3e84428d-d98c-413b-88d3-7fbcaa48e1ef)

![image](https://github.com/Skyrat-SS13/Skyrat-tg/assets/102194057/e1923aea-78cb-42a8-a69d-a1510dac3ab0)

![image](https://github.com/Skyrat-SS13/Skyrat-tg/assets/102194057/153cb6fc-ca63-41bf-b5f5-99e5172c4bc4)

![image](https://github.com/Skyrat-SS13/Skyrat-tg/assets/102194057/b48759c4-141b-4ae3-b78c-4841c52a4c2b)

![image](https://github.com/Skyrat-SS13/Skyrat-tg/assets/102194057/ec40f103-56e7-4b4d-891f-95aef46b77ad)

![image](https://github.com/Skyrat-SS13/Skyrat-tg/assets/102194057/a6eeeb24-0fef-4689-aca6-d2baef9286b0)

![image](https://github.com/Skyrat-SS13/Skyrat-tg/assets/102194057/84ac5860-a612-4af3-a3f4-a00e30a73d26)

![image](https://github.com/Skyrat-SS13/Skyrat-tg/assets/102194057/138a8061-a570-4ef2-884c-602f26ca6592)

![image](https://github.com/Skyrat-SS13/Skyrat-tg/assets/102194057/f66b04a2-765d-4316-bed8-74e44be8daab)

</details>

## Changelog

<!-- If your PR modifies aspects of the game that can be concretely observed by players or admins you should add a changelog. If your change does NOT meet this description, remove this section. Be sure to properly mark your PRs to prevent unnecessary GBP loss. You can read up on GBP and it's effects on PRs in the tgstation guides for contributors. Please note that maintainers freely reserve the right to remove and add tags should they deem it appropriate. You can attempt to finagle the system all you want, but it's best to shoot for clear communication right off the bat. -->

:cl:
add: Added and moved some areas in the ghost cafe.
add: Added an icewalker section to the ghost cafe.
add: Added an ashwalker section to the ghost cafe.
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

sqnztb commented:
Okay. A few... issues.

![image](https://github.com/Skyrat-SS13/Skyrat-tg/assets/28007787/fff23e7d-a96f-4932-ad33-b3f9724772e7)
The ID console is a full console, and cafe visitors spawn with with access. Let's not have that.

![image](https://github.com/Skyrat-SS13/Skyrat-tg/assets/28007787/01775bc8-8f31-41bf-978b-7d2c78b8b6f0)
Engineering radio headset in the locker. Also bad.

![image](https://github.com/Skyrat-SS13/Skyrat-tg/assets/28007787/caa87ccd-6a01-4ba5-8eb6-f22ab45e60b8)
Hidden beach cabin is dark. 

![image](https://github.com/Skyrat-SS13/Skyrat-tg/assets/28007787/03f73e1f-c00a-4565-a90c-d58ff490735a)
Another ID console. Power monitoring console which does nothing. I'll also point out at this point that ALL OF THESE consoles (including the civilian ones scattered around) all have NT Software Hub and could download things like the **science hub**, chat client, bounty board requests, etc. More cafe-station interaction points that we don't want.

![image](https://github.com/Skyrat-SS13/Skyrat-tg/assets/28007787/3dd158db-8c63-4801-a5e3-89602a4f5ed7)
I know this is an issue on the live server (which I'll probably fix in a few minutes), but paintings need to be moved off of the wall tiles and pixel-shifted properly (so they don't show through the walls).

![image](https://github.com/Skyrat-SS13/Skyrat-tg/assets/28007787/fe7697e5-70a4-4c9c-8377-05f9ab045f83)
Weird transition here.

![image](https://github.com/Skyrat-SS13/Skyrat-tg/assets/28007787/0a165099-26fa-4ec9-9c91-5ea836456864)
This looks awkward since you moved the wall up but didn't shift the shower up.

Vox box still isn't safe for vox.

Personal gripe, but stepping out of the shuttle and into a waiting area that looks like medbay feels awkward.

The ashwalker camp is cool. The icecat camp is nice, though I don't approve of the pile of surgery tools laying on the rack. I appreciate a lot of the effort that went into doing this, but I don't really think the mini-station is needed, at least in the current form, and some of the moves of some of the existing rooms make for some really awkward spots like - 
![image](https://github.com/Skyrat-SS13/Skyrat-tg/assets/28007787/c8caf07c-5935-4abf-9f7b-b8db3f03ff0f)

Thlumyn commented:
Thanks for looking at it @sqnztb and thanks for the quick response!

I've made changes to those things you have pointed out. 

I added fake consoles based on the fakeid and fakesec consoles from the showcase. I completely forgot it was possible to add more programs onto civilian consoles, o o p s. 

I changed the voxbox to nitrogen atmos. 

As far as that entryway - it was supposed to be "clean white aesthetic" with blue trim for NT, but I see totally how it looks like medbay now that you point it out, I darkened the floor tiles to match the NT area to the left. 

As far as the existence of the ministation and the placement of the old cabins go, that's for @TheSS13Melon  to respond.

Thlumyn commented:
Oops I missed a computer

YakumoChen commented on modular_skyrat/modules/ghostcafe/code/ghost_role_spawners.dm:
```diff
@@ -28,8 +28,8 @@
 		new_spawn.AddElement(/datum/element/dusts_on_catatonia)
-		new_spawn.AddElement(/datum/element/dusts_on_leaving_area,list(A.type, /area/misc/hilbertshotel, /area/centcom/holding/cafe, /area/centcom/holding/cafewar, /area/centcom/holding/cafebotany,
-		/area/centcom/holding/cafebuild, /area/centcom/holding/cafevox, /area/centcom/holding/cafedorms, /area/centcom/holding/cafepark, /area/centcom/holding/cafeplumbing))
+		new_spawn.AddElement(/datum/element/dusts_on_leaving_area,list(A.type, /area/misc/hilbertshotel, /area/centcom/holding/cafe,
```
Hilberts Hotel is a valid location reachable by normal players through a space ruin, I do not recommend including it here

Thlumyn replied:
That was already there, I didn't mess with that.
The ghost cafe already has access to hilbert's hotel, and a ton of people use the hotel rooms there for scenes, so we wouldn't be removing that.

Thlumyn commented:
Well, if Deadmon doesn't respond to that soon, I might just go in and make some changes myself.

sqnztb commented:
Much, much, much better looking.

Okay. Couple of tiny requests.

![image](https://github.com/Skyrat-SS13/Skyrat-tg/assets/28007787/cd40555f-9191-444a-9f5a-4ca6cf368736)
Give a button for curtain toggling. Just makes it a bit of a hassle to close them all otherwise.

![image](https://github.com/Skyrat-SS13/Skyrat-tg/assets/28007787/46163599-bc6f-4c64-a661-c5f1a22129fe)
It's probably okay to leave this as an actual surgery computer? We've got one already in the cafe. Maybe find a place to shove a NanoMed, NanoDrug Plus, and a few medkits like we have currently. I see no harm in leaving them as a *just in case* situation.

Everything else looks good to go as far as I'm concerned.

Thlumyn commented:
Okay @sqnztb and @TheSS13Melon  I have made some changes to make it a bit more cafe-like 

I condensed medbay and robotics together into a "Healthcare Center".
![image](https://github.com/Skyrat-SS13/Skyrat-tg/assets/102194057/ae601a73-4553-422c-a275-2e7748a42b4c)

I removed the bridge and condensed security down into one room and one cell named "Cafe Security" and also added a little river in the now empty area.
![image](https://github.com/Skyrat-SS13/Skyrat-tg/assets/102194057/992edb27-8bfc-448c-8915-816e2658228a)

I removed the engineering section and replaced it with a spa that includes a "hot tub", barber area and massage area. I don't know, if I were on vacation in a cafe resort that's what I'd want to see. I also renamed the generic command office into "Cafe Manager Office". I suppose that way its more cafe immersive but people can still use it for those head of staff office RP if they want. 
![image](https://github.com/Skyrat-SS13/Skyrat-tg/assets/102194057/d0e8ff90-da0e-4be4-9b63-14266ba5745e)

I moved around some of the cabins that had been around just slightly - mostly this one, and added a little patio. To try and make that hallway less clunky. 
![image](https://github.com/Skyrat-SS13/Skyrat-tg/assets/102194057/02445afe-6095-4cba-b685-930c2fecb7fb)

Also fixed some of the paintings while I was at it, replaced the photocopier with a free one, and added a towel bin near the pool.

Thlumyn commented:
Oh oops, I didn't see your response, I accidentally left the comment open but unsent.

Thlumyn commented:
Okay, I added the curtains (the surgery computer, medkits and medical vendors)
![image](https://github.com/Skyrat-SS13/Skyrat-tg/assets/102194057/0c691911-1dac-4b40-939a-a6fc03807afb)

theselfish commented:
![image](https://github.com/Skyrat-SS13/Skyrat-tg/assets/102828457/5030c8e0-8d33-4eb2-8110-d201944f4198)
You seem to have left a bench in the room near botany.
![image](https://github.com/Skyrat-SS13/Skyrat-tg/assets/102828457/2d6bf6d6-0ac2-466d-b184-3e531084d8b7)
If this is meant to be a grilling pit, you should make it the grille version.

SomeRandomOwl commented:
![dreamseeker_GLAhjg2i3X](https://github.com/Skyrat-SS13/Skyrat-tg/assets/2568378/10b0f292-d0ce-4960-bb37-e596efb9a41f)

![dreamseeker_UZXZW4WPLy](https://github.com/Skyrat-SS13/Skyrat-tg/assets/2568378/bc3ae0a5-1016-447c-a48b-9aa031aaad39)

The news caster should be removed and that civilian console should be replaced with a fake. as they both enable station interaction

Thlumyn commented:
Thanks, I didn't even see those.

Gandalf2k15 reviewed (approved):
Looks great!
